tests/runtime/builtin

```diff
@@ -264,3 +264,82 @@ NAME usermode_builtin_in_unsupported_arch
 PROG begin { printf("%d ", usermode);  } end { print(usermode); }
 EXPECT ERROR: 'usermode' builtin is only supported on x86_64
 ARCH aarch64
+
```

Comment:
Did you want to put this stuff into `macro`? It seems more an implementation detail of that than the specific builtins.